### DISCUSSION TITLE
Meeting Notes

### DISCUSSION THREAD

**Discussion by MarioShontz** [emelex-ai/BRIDGE#75] [General]
Issues:
- README issues (Pramodh)
- streamlining of dependency/environment management via Poetry (pyproject.toml) and Dockerfile (and removal of need for requirements.txt).

Assignment for next meeting:
- Run code locally
- make a change in the model (correspondent to your role) and run it again (maybe try to interpret it if its meaningful).
- Get comfortable with workflow: Issue Creation, Linked Branches, pull requests, etc.

**Comment by mathnathan** [comment]
Great writeup, @MarioShontz! Thanks. 

Please reference this to start the next assignment: @pramodhfsu, @abdulrazzak002, @gmartini2019, @Madtrix-777. 

If you have high-level topics you'd like to discuss related to the meeting or the assignment, let's chat about them here. Once you have something specific you'd like to work on, create a more specific discussion topic or jump straight into creating the issue. I look forward to getting started on this with all of you!
